src/Temporalio/Client/TlsOptions.cs

```diff
@@ -34,6 +34,11 @@ public class TlsOptions : ICloneable
         /// </remarks>
         public byte[]? ClientPrivateKey { get; set; }
 
+        /// <summary>
+        /// Gets or sets BLAH BLAH.
```

Comment:
Blah?

Reply:
Thanks! I was testing the acronym fixer and missed this.